packages/contracts/evm-contracts/contracts/token/IInverseAppProjectedNft.sol

```diff
@@ -3,14 +3,14 @@ pragma solidity ^0.8.13;
 
 import {IInverseProjectedNft} from "./IInverseProjectedNft.sol";
 
-/// @dev A standard ERC721 that accepts calldata in the mint function for any initialization data needed in a Paima dApp.
+/// @dev A standard ERC721 that can be freely minted and stores an unique <minter, userTokenId> pair (used in tokenURI) when minted.
```

Comment:
Do we have to update these comments in PRC3 as well?

Reply:
Yes good point, I've updated it here and also there.